components/dashboard/src/settings/settings-menu.ts

```diff
@@ -29,6 +30,14 @@ export default function getSettingsMenu(params: { userBillingMode?: BillingMode
             link: [settingsPathNotifications],
         },
         ...renderBillingMenuEntries(params.userBillingMode),
+        ...(params.showPersonalAccessTokens
```

Comment:
Noob question, can we not use the FeatureFlagContext directly in this method to avoid having to pass it in as an argument?

Reply:
What prevents it from being a React file? I'm fine with this change, just trying to understand the reasoning a bit better.

Reply:
Hmm, I think when we know we're not returning JSX or HTML, there is no need to have something be a React component. But I actually wondered why the feature flag needed to be a Context 🤔